clippy_lints/src/methods/or_fun_call.rs

```diff
@@ -86,6 +86,8 @@ pub(super) fn check<'tcx>(
         span: Span,
         // None if lambda is required
         fun_span: Option<Span>,
+        // `Some` if fn has second argument
+        second_arg: Option<&hir::Expr<'_>>,
```

Comment:
Please place this argument before `span` and `fun_span`.

Reply:
I'll fix it. Shall I squash these fix commits?

Reply:
Yes, that would be great. Will merge right after that.